### PR TITLE
Create a User table with Sequelize

### PR DESCRIPTION
**Pull request by merecka**
Used Sequelize to created a User model & table in Postgres database.

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require("dotenv").config();
+module.exports = {
```
Comment (ahaly92) [inline config/config.js]: it's generally easier to merge these into on string and have this file look like 

`module.exports = {
  development: {
    use_env_variable: 'SQL_CONNECTION_URL',
  },
  production: {
    use_env_variable: 'SQL_CONNECTION_URL',
  },
}
`

File: models/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+const env = process.env.NODE_ENV || "development";
+const config = require(__dirname + "/../config/config.js")[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(
+    config.database,
```
Comment (ahaly92) [inline models/index.js]: sequelize also supports taking a connection string url like postgres://username:password@hosturl/database

Comment (ahaly92) [reply models/index.js]: i think instead of having 5 environment variables i'd prefer to consolidate

File: models/index.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+const fs = require("fs");
+const path = require("path");
+const Sequelize = require("sequelize");
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || "development";
+const config = require(__dirname + "/../config/config.js")[env];
+const db = {};
+
+let sequelize;
```
Comment (ahaly92) [inline models/index.js]: can we move all the sequelize logic in a function in a file called sequelize.js that we can call in here?

File: config/config.js (hunk 1 above)
Comment (merecka) [reply config/config.js]: Done

File: models/index.js (hunk 2 above)
Comment (merecka) [reply models/index.js]: Done

Comment (ahaly92) [reply models/index.js]: const sequelize = new Sequelize(process.env.SQL_CONNECTION_URL, config);

**Pull request merged**